src/Tribe/Template.php

```diff
@@ -872,22 +872,30 @@ public function template( $name, $context = [], $echo = true ) {
 		 */
 		do_action( 'tribe_template_before_include', $file, $name, $this );
 
-		/**
-		 * Fires an Action for a given template name before including the template file
-		 *
-		 * E.g.:
-		 *    `tribe_template_before_include:events/blocks/parts/details`
-		 *    `tribe_template_before_include:events/embed`
-		 *    `tribe_template_before_include:tickets/login-to-purchase`
-		 *
-		 * @deprecated   4.11.0
-		 * @since  4.7.20
-		 *
-		 * @param string $file      Complete path to include the PHP File
-		 * @param array  $name      Template name
-		 * @param self   $template  Current instance of the Tribe__Template
-		 */
-		do_action( "tribe_template_before_include:$legacy_hook_name", $file, $name, $this );
+		if ( $legacy_hook_name !== $hook_name ) {
+			/**
+			 * Fires an Action for a given template name before including the template file
+			 *
+			 * E.g.:
+			 *    `tribe_template_before_include:events/blocks/parts/details`
+			 *    `tribe_template_before_include:events/embed`
+			 *    `tribe_template_before_include:tickets/login-to-purchase`
+			 *
+			 * @since        4.7.20
+			 *
+			 * @deprecated   4.11.0
+			 *
+			 * @param string $file     Complete path to include the PHP File
+			 * @param array  $name     Template name
+			 * @param self   $template Current instance of the Tribe__Template
+			 */
+			do_action_deprecated(
```

Comment:
Do we have an idea about how old is *the most recent code* using the legacy hook names?
If that code is not, at least, 1.5yrs old, then I would add a todo task and not throw a deprecation notice for now.

Reply:
The hooks were added in TEC 5.0.0, so not 1.5 years old?

Reply:
So I would not deprecate them yet.